bin/pfcmd.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -T
+#!/usr/bin/perl
```

Comment:
I was trying to figure out what is causing the issue.
It is not an issue Module::Pluggable since I am able to replicate this without it.

However it may not be worth trying to fix even though I would prefer to at this point and time.
Since it seems to be a very random problem.
We have a straightforward solution at the moment.